core/src/components/searchbar/searchbar.scss

```diff
@@ -64,3 +64,14 @@ ion-searchbar {
 .searchbar-has-value.searchbar-has-focus .searchbar-clear-icon {
   display: block;
 }
+
+// adding rtl support 
+[dir="rtl"]  .searchbar-search-icon {
+  left: auto;
+  right: 16px;
+}
+
+[dir="rtl"] .searchbar-input{
+  direction: rtl;
+}
```

Comment:
As a standard, we avoid this kind of input behavior for inputs. instead, on the template (of the searchbar, on the input) add `dir="auto"`

Reply:
```
[dir="rtl"] .searchbar-input{
  direction: rtl;
}
```
for some reason it not working correctly 
example 
http://joxi.ru/E2p9vW4TBR1ZPA
after force with above code 
http://joxi.ru/bmo9o64TMVWKKA

Reply:
That example is actually a browser bug - placeholders on `auto` inputs are always on the left.